bin/gsmget

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Tiny speed improvement: this can just be `#!/bin/sh`.

Reply:
sounds great, will tweak.